pywinauto/recorder/win32/win32_recorder.py

```diff
@@ -0,0 +1,216 @@
+import threading
+import ctypes
+import timeit
+import time
+
+from ..recorder_defines import EVENT, PROPERTY
+
+from ... import win32_hooks
+from ... import win32defines
+from ... import handleprops
+from ...win32structures import POINT
+from ...win32_element_info import HwndElementInfo
+from ...controls.hwndwrapper import InvalidWindowHandle
+from win32api import LOWORD, HIWORD
+import win32gui
+
+from ..uia.uia_recorder import ProgressBarDialog
+from ..control_tree import ControlTree
+from ..base_recorder import BaseRecorder
+from ..recorder_defines import RecorderEvent, \
+    RecorderKeyboardEvent, \
+    RecorderMouseEvent, \
+    ApplicationEvent, \
+    PropertyEvent
```

Comment:
One name for import should be at one line. See PEP8 #imports: https://www.python.org/dev/peps/pep-0008/#imports